crypto/bio/bss_sock.c

```diff
@@ -111,7 +128,23 @@ static int sock_write(BIO *b, const char *in, int inl)
     int ret;
 
     clear_socket_error();
-    ret = writesocket(b->num, in, inl);
+    if (BIO_should_ktls_ctrl_msg_flag(b)) {
+        unsigned char record_type = (intptr_t)b->ptr;
+
+#ifdef SSL_DEBUG
+        printf("\nsending ctrl msg\n");
+#endif
+        ret = ktls_send_ctrl_message(b->num, record_type, in, inl);
```

Comment:
Needs an `OPENSSL_NO_KTLS` guard. This is the cause of the Appveyor failure.

Reply:
Done.